clients/StateChannelWallet.ts

```diff
@@ -175,13 +175,10 @@ export class StateChannelWallet {
    * getBalance checks the blockchain for the current balance of the wallet.
    */
   async getBalance(): Promise<number> {
-    return 10;
-    // todo: caching, block event based updating, etc
-    // const balance = await this.chainProvider.getBalance(
-    //   this.scBridgeWalletAddress,
-    // );
-    // const balanceEther = ethers.formatEther(balance);
-    // return Number(balanceEther);
+    // TODO: Casting a bigint to a number is dangerous
```

Comment:
We could just return the bigint and let the UI handle displaying it?

Another idea would be to avoid using ETH (very large amount of wei).